### ISSUE TITLE
[arcgis-rest-types] IStatisticDefinition.statisticParameter not matching JSAPI statisticParameters

### ISSUE THREAD

**Issue by pemberdom** [Esri/arcgis-rest-js#692]
The interface `IStatisticDefinition` has a `statisticParameter` property that doesn't match with the JSAPI (and REST API) `statisticParameters`.

The "s" might have been lost during [this discussion](https://devtopia.esri.com/WebGIS/arcgis-js-api/issues/24952#issuecomment-2036517).

The consequence is that a conversion from `IStatisticDefinition` to `StatisticDefinition` does not translate the `statisticParameter(s)` property:
```ts
const stat: IStatisticDefinition = {
  // ...
  statisticParameter: { ... };
};

const convertedStat = StatisticDefinition.fromJSON(stat); // does not contain a value for `statisticParameters`
```

Could a fix been brought to make that conversion possible ? Thanks !

**Comment by pemberdom** [comment]
cc @maneetgoyal @subgan82

**Comment by subgan82** [comment]
@pemberdom can we close the issue , it is now available in [2.12.1](https://github.com/Esri/arcgis-rest-js/pull/696#issuecomment-623552969)
